Src/ILGPU.Algorithms/Random/RNG.cs

```diff
@@ -158,17 +158,17 @@ protected RNG(Accelerator accelerator)
         /// <summary>
         /// The maximum number of parallel groups.
         /// </summary>
-        private readonly int groupSize;
+        private readonly int maxNumParallelWarps;
 
         /// <summary>
         /// Initializes the RNG view.
         /// </summary>
         /// <param name="providers">The random providers.</param>
-        /// <param name="numParallelGroups">The maximum number of parallel groups.</param>
-        internal RNGView(ArrayView<TRandomProvider> providers, int numParallelGroups)
+        /// <param name="numParallelWarps">The maximum number of parallel groups.</param>
```

Comment:
MINOR: Comment not updated.
```suggestion
        /// <param name="numParallelWarps">The maximum number of parallel warps.</param>
```